src/opnsense/mvc/app/views/OPNsense/Firewall/alias.volt

```diff
@@ -87,9 +87,9 @@
                     },
                     timestamp: function (column, row) {
                         if (row[column.id] && row[column.id].includes('.')) {
-                            return row[column.id].split('.')[0];
+                            return row[column.id].split('.')[0].replace('T', ' ');
                         }
-                        return row[column.id];
+                        return row[column.id].replace('T', ' ');;
```

Comment:
@NOYB this could be null, better leave this line as is, don't mind replacing the `T` when we know it's a time.